tutorials/asr/06_Voice_Activiy_Detection.ipynb

```diff
@@ -27,7 +27,7 @@
     "!pip install unidecode\n",
     "\n",
     "# ## Install NeMo\n",
-    "BRANCH = 'v1.0.0b2'\n",
+    "BRANCH = 'v1.0.0b3'\n",
```

Comment:
@fayejf should VAD tutorial be `v1.0.0b3` or `main`?

Reply:
Let's use main. If that's not good I will change it to 1.0.0b3. Thanks